### PR TITLE
Duplicate jobs for Bullseye

### PR DESCRIPTION
**Pull request by sopelj**

### CODE REVIEW

File: zuul.d/wazo_nodesets.yaml [hunk 1]
```diff
@@ -41,8 +41,20 @@
       - label: vm-debian-10-m2s
         name: vm-debian-10-m2s
 
+- nodeset:
+    name: vm-debian-11-m2s
+    nodes:
+      - label: vm-debian-11-m2s
+        name: vm-debian-11-m2s
```
Comment (fblackburn1) [inline zuul.d/wazo_nodesets.yaml:48]: What is the purpose of this job? Is it used by someone?

Comment (sopelj) [reply zuul.d/wazo_nodesets.yaml:48]: You're correct. It doesn't seem to be used. The only reference I can find is in `wikijs-storage-production/infrastructure/software_factory.md`. So, I'll remove the debian-11 one and we might even be able to remove the debian-10 one.

File: zuul.d/wazo_jobs.yaml [hunk 2]
```diff
@@ -128,3 +148,17 @@
...
+- job:
+    name: wazo-ansible-uc-bullseye
+    description: Install Wazo Platform UC use case
+    timeout: 10800
+    pre-run: playbooks/wazo-ansible/pre.yaml
+    run: playbooks/wazo-ansible/run.yaml
+    post-run: playbooks/wazo-ansible/fetch-logs.yaml
+    vars:
+      wazo_debian_repo: main
+      wazo_distribution: wazo-dev-buster
```
Comment (fblackburn1) [inline zuul.d/wazo_jobs.yaml:161]:
```suggestion
      wazo_distribution: wazo-dev-wip-bullseye
```

**Review by fblackburn1 — COMMENTED** [review]
We will need to add task in our list to remove (or rename) the `wazo-dev-wip-bullseye`
Maybe also remove all `bullseye` suffix after migration?

**Review by fblackburn1 — APPROVED** [review]

**Pull request merged**